src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/Folders/AddFolder.cs

```diff
@@ -0,0 +1,26 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.SqlProjects.Contracts
+{
+    /// <summary>
+    /// Parameters for adding, deleting, or excluding a folder
+    /// </summary>
+    public class FolderParams : SqlProjectParams
+    {
+        /// <summary>
+        /// Path of the folder, typically relative to the .sqlproj file
+        /// </summary>
+        public string Path { get; set; }
+    }
+
+    public class AddFolderRequest
+    {
+        public static readonly RequestType<FolderParams, ResultStatus> Type = RequestType<FolderParams, ResultStatus>.Create("sqlprojects/addFolder");
```

Comment:
Feel free to capitalize the 'p' in "sqlprojects/..." here too.

Reply:
Converted 'em all!